utils/data-utils/src/index.js

```diff
@@ -32,7 +32,7 @@ const formatLink = (l) => {
     link.link.hreflang = l.hreflang;
   }
 
-  link.link.aria_label = l.ariaLabel ? l.ariaLabel : 'Aria label default';
+  link.link.aria_label = l.ariaLabel ? l.ariaLabel : '';
```

Comment:
Not an issue, though when you have a check on a thing you want to either use or put a default value you could do as
```
  link.link.aria_label = l.ariaLabel || '';
```
Simpler to read and understand putting a default value.